### PR TITLE
Updates Discord sending mechanism to use embed.

### PR DESCRIPTION
**Pull request by Evo-F**
Secondary PR, replaces PR #1

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -155,9 +171,13 @@ let lastMessageTime = new Date();
 let queuedMessagesPrivate = [];
 let queuedMessagesPublic = [];
 let message = '';
+let clearMessagesPrivate = false;
+let clearMessagesPublic = false;
```
Comment (devnull9090) [inline main.js]:
```suggestion
```

File: main.js [hunk 2]
```diff
@@ -155,9 +171,13 @@ let lastMessageTime = new Date();
...
 
 
 function updateSubreddit(data, _new = false) {
+    clearMessagesPrivate = false;
+    clearMessagesPublic = false;
```
Comment (devnull9090) [inline main.js]:
```suggestion
```

File: main.js [hunk 3]
```diff
@@ -201,14 +221,16 @@ function updateSubreddit(data, _new = false) {
                 message += "\n";
             }
             message += "Subreddits that have gone dark: " + queuedMessagesPrivate.join(", ");
-            queuedMessagesPrivate = [];
+            clearMessagesPrivate = true;
+            // queuedMessagesPrivate = [];
```
Comment (devnull9090) [inline main.js]:
```suggestion
```

File: main.js [hunk 4]
```diff
@@ -201,14 +221,16 @@ function updateSubreddit(data, _new = false) {
...
         }
         if (queuedMessagesPublic.length > 0) {
             if (message.length > 0) {
                 message += "\n";
             }
             message += "Subreddits that have gone public: " + queuedMessagesPublic.join(", ");
-            queuedMessagesPublic = [];
+            clearMessagesPublic = true;
+            // queuedMessagesPublic = [];
```
Comment (devnull9090) [inline main.js]:
```suggestion
```

**Review by devnull9090 — CHANGES_REQUESTED** [review]

**Comment by Evo-F** [comment]
image

**Review by devnull9090 — APPROVED** [review]

**Comment by devnull9090** [comment]
Thank you for the contribution!

**Pull request merged**